### PR TITLE
Fix gulp watch task

### LINKED ISSUE #101: Dev server not working
The development server doesn't live reload on changes and doesn't automatically build on CSS changes.

### PR DESCRIPTION
**Pull request by seriouslysean**
It looks like we just weren't watching for updates on the partial files so I added a new `styles.watch` variable to utilize in the watch task. Also commented the general flow to help with anyone else that'll be editing this in the future 👍 

Closes #101

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -115,18 +115,19 @@ function style() {
...
+  // Run all required tasks once
+  style();
 
-  gulp.watch(paths.srcDir, style)
-  gulp.watch([paths.srcDir, paths.docsDir], browserSync.reload)
+  // Watch files for changes to reload when detected
+  gulp.watch([paths.docsDir, paths.styles.watch]).on('change', gulp.series(style, browserSync.reload));
 }
 
-module.exports.style = style
```
Comment (seriouslysean) [inline gulpfile.js]: I did a cursory search and didn't see this being used anywhere but I'm now seeing the deployment tests failing. If this was needed somewhere I didn't notice, let me know and I'll put it back.

**Comment by kylejrp** [comment]
That line is used by the command `yarn build` in the `package.json` file. It's necessary to create the build files and that's probably why the deployment is failing. Would you be able to modify your commits to undo this change?

I'll try to review this when I can!

File: gulpfile.js (hunk 1 above)
Comment (seriouslysean) [reply gulpfile.js]: @kognise added it back, thanks for the feedback.

File: gulpfile.js [hunk 2]
```diff
@@ -57,75 +57,77 @@ function style () {
   return (
     gulp
       .src(paths.styles.src)
-      // Add sourcemaps
```
Comment (seriouslysean) [inline gulpfile.js]: Series of linting adjustments 🙃

**Comment by kylejrp** [comment]
Awesome, thanks @seriouslysean. I'll review and merge when I can.

**Comment by kognise** [comment]
Gonna work on manually merging this into the 2.0 branch now.

**Pull request closed without merging**